### PR TITLE
Idea for GSoC: Implementing CHAOSS metrics with Perceval

### PR DESCRIPTION
**Pull request by jgbarah**
This is a proposal for GSoC'2019.

### CODE REVIEW

File: GSoC-Ideas.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+documentation.
+
+  * _Difficulty:_ Medium
+  * _Requirements:_ Python programming, experience with Python notebooks, skills for producing documentation.
+  * _Recommended:_ Experience in data analytics with Python, if possible involving the use of Pandas, will be a plus. 
+  * _Mentors:_ Jesus M. Gonzalez-Barahona and Valerio Cosentino.
+
+\[Below, template for proposing ideas. Pull request with your idea. \]
+
+### Idea #x:
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: Please remove the duplicate idea template (lines 43-60)

File: GSoC-Ideas.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+The aims of this idea are as follows:
+
+* Producing Python (Jupyter) notebooks with proposals for reference implementations
+for these metrics. The proposals will follow the usual acceptance mechanism of the working group
+(based on GitHub issues and pull requests). In general, all reference implementations will
+build on the output produced by Perceval for the relevant data source,
+and will explore the peculiarities of the metric and its implementation for that data source.
+
+* Document the notebooks as much as possible, so that any person trying to implement the metric
+can understand not only how to implement it, but also the details that should have into account.
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: line 25: This sentence I don't understand; it seems incomplete: "but also the details that should have into account."

File: GSoC-Ideas.md (hunk 1 above)
Comment (jgbarah) [reply GSoC-Ideas.md]: I did. Thanks for noticing.

File: GSoC-Ideas.md (hunk 2 above)
Comment (jgbarah) [reply GSoC-Ideas.md]: My bad. Is it more clear now?

**Pull request merged**